### PR TITLE
fix: add NUTs

### PR DESCRIPTION
**Pull request by WillieRuemmele**
### What does this PR do?
adds NUTs covering each command
### What issues does this PR fix or reference?
@W-8856601@

### CODE REVIEW

File: test/commands/alias/set.nut.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+      expect(getNumber(res.jsonOutput, 'status')).to.equal(1);
+      const list = execCmd('alias:list --json', {
+        ensureExitCode: 0,
+      });
+      expect(list.jsonOutput).to.deep.equal({
+        result: [],
+        status: 0,
+      });
+    });
+  });
```
Comment (shetzel) [inline test/commands/alias/set.nut.ts:121]: Same comment for this nut.

File: test/commands/alias/unset.nut.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        status: 0,
+        result: [
+          {
+            alias: 'Admin',
+            value: 'admin@salesforce.com',
+          },
+        ],
+      });
+    });
+  });
```
Comment (shetzel) [inline test/commands/alias/unset.nut.ts:114]: same comment

**Review by shetzel — CHANGES_REQUESTED** [review]
The reason for request changes is to add at least 1 non-JSON test for each command.  Otherwise, 👍

**Review by mshanemc — APPROVED** [review]
seems fine.

The only thing I don't love about it is how many test sessions it uses.

That's not a pattern we want people to use in general (because of perf) but it doesn't really matter for `alias` when there's no orgs/projects.

**Comment by WillieRuemmele** [comment]
@shetzel @mshanemc I've added non-JSON tests and reduced the amount of TestSessions I'm using

File: test/commands/alias/list.nut.ts [hunk 3]
```diff
@@ -7,12 +7,20 @@
...
+
+function unsetAll() {
+  execCmd('sfdx alias:unset DevHub');
+  execCmd('sfdx alias:unset Admin');
+  execCmd('sfdx alias:unset user');
+}
+
 describe('alias:list NUTs', () => {
-  let testSession: TestSession;
+  testSession = TestSession.create({});
```
Comment (shetzel) [inline test/commands/alias/list.nut.ts:19]: nit: just make this `const testSession = TestSession.create();` and remove the `let testSession: TestSession;` above

File: test/commands/alias/list.nut.ts [hunk 4]
```diff
@@ -43,20 +55,24 @@ describe('alias:list NUTs', () => {
...
-          'sfdx alias:set DevHub=mydevhuborg@salesforce.com',
-          'sfdx alias:set Admin=admin@salesforce.com',
-          'sfdx alias:set user=user@salesforce.com',
-        ],
-      });
+    beforeEach(() => {
+      unsetAll();
+      execCmd('alias:set DevHub=mydevhuborg@salesforce.com');
+      execCmd('alias:set Admin=admin@salesforce.com');
+      execCmd('alias:set user=user@salesforce.com');
```
Comment (shetzel) [inline test/commands/alias/list.nut.ts:72]: jsyk, you should be able to set and unset multiple aliases in 1 command

**Review by shetzel — APPROVED** [review]

File: test/commands/alias/list.nut.ts (hunk 4 above)
Comment (RodEsp) [reply test/commands/alias/list.nut.ts:72]: But is sacrificing the readability worth it?

**Pull request merged**